### PR TITLE
Support training models with just two classes

### PR DESCRIPTION
**Pull request by pjbull**
Top K accuracy fails to calculate if the number of classes is 2 or fewer. In that case, we just calculate accuracy instead.

### CODE REVIEW

File: tests/conftest.py [hunk 1]
```diff
@@ -78,6 +79,21 @@ def labels_relative_path() -> os.PathLike:
     return ASSETS_DIR / "labels.csv"
 
 
+def labels_n_classes_df(n_classes):
+    """Get up a labels data frame where the labels are up to
```
Comment (ejm714) [inline tests/conftest.py:83]:
```suggestion
    """Get up a labels dataframe where the labels are up to
```

File: tests/conftest.py [hunk 2]
```diff
@@ -78,6 +79,21 @@ def labels_relative_path() -> os.PathLike:
...
+    26 classes.
+    """
+    if n_classes > len(string.ascii_uppercase):
+        raise ValueError("n_classes must be less than 26")
```
Comment (ejm714) [inline tests/conftest.py:87]: what's the point of this?

**Review by ejm714 — APPROVED** [review]
LGTM, just wondering what's magic about 26

File: tests/conftest.py (hunk 2 above)
Comment (pjbull) [reply tests/conftest.py:87]: just cuz it's easy to implement guaranteed unique classes by just using the uppercase letters.

Comment (ejm714) [reply tests/conftest.py:87]: ah make sense

**Pull request merged**